pulpcore/setup.py

```diff
@@ -4,14 +4,14 @@
     long_description = f.read()
 
 requirements = [
-    'celery',
+    'celery>=4',
     'coreapi',
     'Django>=1.11',
     'django-filter',
     'djangorestframework',
     'drf-nested-routers',
     'drf-yasg',
-    'psycopg2',
```

Comment:
Should attach this issue: https://pulp.plan.io/issues/3375

I also left a comment on that issue with something I noticed on the ```psychopg2-binary``` page.